src/components/workflowEditor/workflowEditor.tsx

```diff
@@ -171,16 +180,20 @@ class WorkflowEdit extends Component<WorkflowEditProps, WorkflowEditState> {
     }
 
     addWebhookCD = (workflowId?: number | string) => {
-        this.props.history.push(`${URLS.APP}/${this.props.match.params.appId}/edit/workflow/${workflowId || 0}/webhook/0/cd-pipeline`)
+        this.props.history.push(
+            `${URLS.APP}/${this.props.match.params.appId}/edit/workflow/${workflowId || 0}/webhook/0/cd-pipeline`,
```

Comment:
Could you please use constant for ci-pipeline & workflow

Reply:
Done